packages/types/src/Metadata/v0/Modules.ts

```diff
@@ -174,21 +213,36 @@ export class MapType extends Struct {
 class PlainType extends Type {
 }
 
-export class StorageFunctionType extends EnumType<PlainType | MapType> {
+export class StorageFunctionType extends EnumType<PlainType | MapType | DoubleMapType> {
```

Comment:
Just to understand your intent here: we agree that v0 doesn't have `DoubleMapType`, but you're still putting it here to be able to factorize code, as this change doesn't break compatibility, correct?

If yes, then see my 2 comments above where we can factorize code a bit more.